src/main.rs

```diff
@@ -13,6 +15,16 @@ const DEFAULT_NATS_HOST: &str = "localhost";
 const DEFAULT_NATS_PORT: &str = "4222";
 /// Default lattice prefix
 const DEFAULT_LATTICE_PREFIX: &str = "default";
+/// Default log level
+const DEFAULT_LOG_LEVEL: &str = "info";
+/// Default allow loading components from the file system
+const DEFAULT_ALLOW_FILE_LOAD: &str = "false";
```

Comment:
Why not `bool`?

Reply:
It's weird, but clap expects something that can be converted into a clap OsStr
```
the trait bound `clap::builder::OsStr: From<bool>` is not satisfied
the following other types implement trait `From<T>`:
  <clap::builder::OsStr as From<&&'static std::ffi::OsStr>>
  <clap::builder::OsStr as From<&&'static str>>
  <clap::builder::OsStr as From<&'static std::ffi::OsStr>>
  <clap::builder::OsStr as From<&'static str>>
  <clap::builder::OsStr as From<&Str>>
  <clap::builder::OsStr as From<&clap::builder::OsStr>>
  <clap::builder::OsStr as From<Str>>
required for `bool` to implement `Into<clap::builder::OsStr>`
```

Reply:
Figured it out. You can use `default_value` for pre-processed args (user input in CLI) and `default_value_t` for post-processed args